tests/regForm_DZ18.spec.js

```diff
@@ -0,0 +1,305 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("Registration form", () => {
+    test.describe("Name field validation", () => {
+        test.beforeEach(async ({ page }) => {
+            await page.goto("https://qauto.forstudy.space/");
+            const signUpBtn = page.locator('.btn-primary', {hasText: 'Sign Up'} );
+            await signUpBtn.click();
+        });
+
+        test("Positive test - valid name input", async ({ page }) => {
+            const nameInput = page.locator('#signupName');
+            const validName = "Will";
+
+            await nameInput.fill(validName);
+            await nameInput.press("Enter");
+
+            await expect(nameInput).not.toHaveClass("invalid");
```

Comment:
use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-css

Reply:
you should check border color in every negative scenario